wifi_manager/core.py

```diff
@@ -194,13 +193,14 @@ def do_nothing():
     raise WifiException(e.message, 500)
 
 
-def delete(iface, ssid, db):
+def delete(iface, ssid, db, db_only=False):
```

Comment:
Let's talk and think about why this flag is needed :)

Reply:
So shall I merge and look later into mocking?